observability/alerts/HCPkasMonitor-prometheusRule_test.yaml

```diff
@@ -0,0 +1,117 @@
+  alert_rule_test:
+  - eval_time: 390m # After 6h+30m to satisfy both windows and 'for' condition (15m)
+    alertname: kas-monitor-ErrorBudgetBurn
+    exp_alerts:
+    - exp_labels:
+        severity: critical
+        long_window: 1h
+        short_window: 5m
+        probe_url: https://api.example.com/health
+        namespace: test-hcp-namespace
+        _id: test-hcp-id
+        cluster: test-cluster
+      exp_annotations:
+        description: "High error budget burn for https://api.example.com/health (current value: 1)"
+    - exp_labels:
+        severity: critical
+        long_window: 6h
+        short_window: 30m
+        probe_url: https://api.example.com/health
+        namespace: test-hcp-namespace
+        _id: test-hcp-id
+        cluster: test-cluster
+      exp_annotations:
+        description: "High error budget burn for https://api.example.com/health (current value: 1)"
+# Test 4: All four alerts should fire simultaneously after prolonged failure
+- interval: 30s
+  input_series:
+  - series: 'probe_success{probe_url="https://api.example.com/health", namespace="test-hcp-namespace", _id="test-hcp-id", cluster="test-cluster"}'
+    # Complete failure for 3+ days to satisfy all four alert time windows
+    values: "0+0x9000" # 4500 minutes = 75 hours = 3+ days of failure
+  alert_rule_test:
+  - eval_time: 4500m # After sufficient time for all windows + 'for' conditions
+    alertname: kas-monitor-ErrorBudgetBurn
+    exp_alerts:
+    - exp_labels:
+        severity: critical
+        long_window: 1h
+        short_window: 5m
+        probe_url: https://api.example.com/health
+        namespace: test-hcp-namespace
+        _id: test-hcp-id
+        cluster: test-cluster
+      exp_annotations:
+        description: "High error budget burn for https://api.example.com/health (current value: 1)"
+    - exp_labels:
+        severity: critical
+        long_window: 6h
+        short_window: 30m
+        probe_url: https://api.example.com/health
+        namespace: test-hcp-namespace
+        _id: test-hcp-id
+        cluster: test-cluster
+      exp_annotations:
+        description: "High error budget burn for https://api.example.com/health (current value: 1)"
+    - exp_labels:
+        severity: critical
+        long_window: 1d
+        short_window: 2h
+        probe_url: https://api.example.com/health
+        namespace: test-hcp-namespace
+        _id: test-hcp-id
+        cluster: test-cluster
+      exp_annotations:
+        description: "High error budget burn for https://api.example.com/health (current value: 1)"
+    - exp_labels:
+        severity: critical
+        long_window: 3d
+        short_window: 6h
+        probe_url: https://api.example.com/health
+        namespace: test-hcp-namespace
+        _id: test-hcp-id
+        cluster: test-cluster
+      exp_annotations:
+        description: "High error budget burn for https://api.example.com/health (current value: 1)"
```

Comment:
Why do last 2 alerting rules have critical severity? AFAIK, 3 and 1 burn rates should just be warnings similar to rules defined in RMO - https://github.com/openshift/route-monitor-operator/blob/master/pkg/alert/prometheusrule.go#L170-L181